Server/JokesOnYou.Web.Api/Services/Interfaces/IJokesService.cs

```diff
@@ -7,5 +8,7 @@ namespace JokesOnYou.Web.Api.Services.Interfaces
 {
     public interface IJokesService
     {
+        public Task<JokeReplyDto> CreateJokeAsync(JokeCreateDto jokeCreateDto, string userId);
```

Comment:
tip: have the userId on the dto (it doesnt matter if someone tries to make a normal api request and sees oo i can enter user id because we will overwrite it with the userid from token)